sentinel-core/src/main/java/com/alibaba/csp/sentinel/node/metric/MetricTimerListener.java

```diff
@@ -25,16 +25,34 @@
 import com.alibaba.csp.sentinel.config.SentinelConfig;
 import com.alibaba.csp.sentinel.log.RecordLog;
 import com.alibaba.csp.sentinel.node.ClusterNode;
+import com.alibaba.csp.sentinel.node.metric.export.DefaultExporter;
+import com.alibaba.csp.sentinel.node.metric.export.MetricExporter;
 import com.alibaba.csp.sentinel.slotchain.ResourceWrapper;
 import com.alibaba.csp.sentinel.slots.clusterbuilder.ClusterBuilderSlot;
+import com.alibaba.csp.sentinel.spi.SpiLoader;
 
 /**
  * @author jialiang.linjl
  */
 public class MetricTimerListener implements Runnable {
 
-    private static final MetricWriter metricWriter = new MetricWriter(SentinelConfig.singleMetricFileSize(),
-        SentinelConfig.totalMetricFileCount());
+    private static MetricExporter exporter = null;
+
+    static {
+        MetricExporter instance = null;
+        try {
+            instance = SpiLoader.of(MetricExporter.class).loadFirstInstance();
```

Comment:
We may need to discuss whether it's more suitable to get the first implementation or to obtain a list to support multiple exporters.